caravel/assets/stylesheets/less/cosmo/bootswatch.less

```diff
@@ -252,18 +248,35 @@ label {
   &-default {
     .panel-heading {
       padding: 15px 15px 0 15px;
+      background-color: transparent;
     }
     .panel-title {
-      font-size: 14px;
-      font-weight: bold;
+      color: black;
+      padding-bottom: 5px;
+      border-bottom: 1px solid @gray-light;
+      h1, h2, h3, h4, h5, h6 {
+        margin: 10px 0 0 0;
+        font-weight: bold;
+      }
+    }
+    .close {
+      color: @text-color;
+    }
+  }
+  &-primary{
```

Comment:
instead of changing these styles can we just change the markup to use`panel-default` rather than`panel-primary` and it should just work without additional css changes.

<img width="387" alt="screenshot 2016-08-29 11 52 32" src="https://cloud.githubusercontent.com/assets/130878/18063162/c68385d8-6dde-11e6-9b43-2f32a9960503.png">